cpp/src/groupby/hash/groupby.cu

```diff
@@ -438,7 +450,10 @@ std::pair<std::unique_ptr<table>, std::vector<aggregation_result>> groupby(
       groupby_null_templated<false>(keys, requests, &cache, include_null_keys, stream, mr);
   }
 
-  return std::make_pair(std::move(unique_keys), extract_results(requests, cache));
+  auto result = extract_results(requests, cache);
+
+  // return std::make_pair(std::move(unique_keys), extract_results(requests, cache));
+  return std::make_pair(std::move(unique_keys), std::move(result));
```

Comment:
What is the reason for this change?

Reply:
That was for debugging something. I will change it back.